Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -157,6 +163,18 @@ - (BOOL)rowAtIndexPathIsOnlyRowVisibleInSection:(NSIndexPath*)indexPath {
     }
 }
 
+/**
+ *  Check whether or not a section is going in or out of view.
+ *
+ *  @param indexPath The index path of the row which will or did end displaying.
+ *
+ *  @return @c YES if that section isn't displaying or the given row is the only one visible in its section, otherwise @c NO.
+ */
+- (BOOL)isVisibilityTransitioningForRowIndexPath:(NSIndexPath*)indexPath {
+    return ![self isDisplayingCellsForSection:indexPath.section]
```

Comment:
Returning `YES` when `![self isDisplayingCellsForSection:indexPath.section]` doesn't seem right. Just because a section is off screen shouldn't mean that it is "transitioning"  - it just means it is off screen. Or am I missing something here?